### PR TITLE
Verbose info queue

### PR DESCRIPTION
**Pull request by gonsie**
Add `-v` and `--verbose` to fsinfo and fsqueue scripts. This just prints the underlying flux command that gets run. I've tested these and they seem to work, but maybe give my perl a double check.

### CODE REVIEW

File: src/fsqueue.py [hunk 1]
```diff
@@ -97,11 +107,16 @@ def main(parsedargs) :
         jobidlist = []
         for j in args.jobs.split(",") :
             jobidlist.append(fjob.id_parse(j))
+            verbose_cmd += j + " "
         mylist = fjob.JobList(myhandle,user=user,ids=jobidlist)
     if args.nodelist:
         myjoblist = filter_byhostlist(mylist, args.nodelist)
+        if args.verbose == True :
+            print("#warning: hostlist filtering not possible with flux-jobs CLI")
```
Comment (alecbcs) [inline src/fsqueue.py:115]: Hi @gonsie! Sweet this is looking good! I'm not sure how we want to standardize warning messages to the user but I might suggest modifying your `#warning: ...` statements to `WARNING: ...` to match consistency within `fsqueue.py`.

Comment (ryanday36) [reply src/fsqueue.py:115]: I think it makes sense to use `WARNING: ...` with anything that we expect to be read by people, but I can see a case for `#warning: ...` for things that you expect to be read by programs. It doesn't look like I've been entirely consistent in other places in the wrappers, but I'd be good with standardizing on `WARNING:`.

**Comment by gonsie** [comment]
thanks for the feedback @ryanday36 and @alecbcs. Please merge this when you are ready.

**Review by ryanday36 — APPROVED** [review]
looks good! Thanks @gonsie.

**Pull request merged**